package.json

```diff
@@ -21,6 +21,7 @@
   },
   "jest": {
     "preset": "ts-jest",
+    "testMatch": ["**/__tests__/**/*test.ts"],
```

Comment:
@l2succes versus move the fixture data file out of the test directory, I added this to ignore files that weren't tests.